src/utils/fs.rs

```diff
@@ -91,15 +91,19 @@ pub fn get_all_files<A: AsRef<Path>>(
         .filter_map(|e| LanguageType::from_path(e.path(), &config).map(|l| (e, l)));
 
     let process = |(entry, language): (DirEntry, LanguageType)| {
-        let result = language.parse(entry.into_path(), &config);
-        let mut lock = languages.lock();
-        let entry = lock.entry(language).or_insert_with(Language::new);
-        match result {
-            Ok(stats) => entry.add_report(stats),
-            Err((error, path)) => {
-                entry.mark_inaccurate();
-                error!("Error reading {}:\n{}", path.display(), error);
+        if ! config.streaming.unwrap_or_default() {
+            let result = language.parse(entry.into_path(), &config);
+            let mut lock = languages.lock();
+            let entry = lock.entry(language).or_insert_with(Language::new);
+            match result {
+                Ok(stats) => entry.add_report(stats),
```

Comment:
The `config.for_each_fn` should be called here before `entry.add_report(stats)`.

Reply:
Thanks for the suggestion, fixed the current implementation accordingly.